test/functional/lua/secure_spec.lua

```diff
@@ -168,4 +168,111 @@ describe('vim.secure', function()
       eq(false, curbufmeths.get_option('modifiable'))
     end)
   end)
+
+  describe('trust()', function()
+    local xstate = 'Xstate'
+
+    setup(function()
+      helpers.mkdir_p(xstate .. pathsep .. (iswin and 'nvim-data' or 'nvim'))
```

Comment:
The linter is complaining here, but I'm not sure why? `iswin` is defined at `:13` and this is 1:1 copy of `:20`.
`/home/runner/work/neovim/neovim/test/functional/lua/secure_spec.lua:176:45: accessing undefined variable iswin`

Reply:
Rebase on master and change `iswin` to `is_os('win')`. This changed recently https://github.com/neovim/neovim/commit/5eb5f4948826e9d47685ea9e257409cc3e693614

Reply:
That was it, idk why I didn't think about it. Thanks.
Done in 73c9b78cde0f5092110b0d053200b7abff52ccb2